### DISCUSSION TITLE
--highlight_words True and --sentence

### DISCUSSION THREAD

**Discussion by Joelson-Forte** [Purfview/whisper-standalone-win#136] [Ideas]
Hi,
I noticed that the "--highlight_words True" option doesn't work when "--sentence" is enabled. Do you think it will one day be possible to overcome this limitation? It would be nice to have these options working together, for example to create the effect of highlighting words in vertical 9x16 video subtitles that require shorter subtitles than the conventional standard. This requires the two options to work together.

Think about it. Thanks again!

**Comment by Purfview** [comment]
--sentence code is too complicated/spaghettified to add the words highlighting, so I won't be doing it.

**Comment by Purfview** [comment]
Maybe I could enable original max_line_width/max_line_count behavior with highlight_words.

**Comment by Purfview** [comment]
Labeled as "implemented" idea as it was more about of `--highlight_words` working with some "max" args of the `--sentence`.
Read there -> https://github.com/Purfview/whisper-standalone-win/discussions/229

Actual --sentence has no effect on highlight_words.
